relay-event-normalization/src/timestamp.rs

```diff
@@ -39,15 +39,45 @@ impl Processor for TimestampProcessor {
             }
         }
 
+        event.process_child_values(self, state)?;
+
+        Ok(())
+    }
+
+    fn process_span(
```

Comment:
Docs for the processor are up-to-date, I accidentally updated them in the previous PR.